packages/app-mobile/components/screens/NoteTagsDialog.js

```diff
@@ -7,7 +7,6 @@ const { _ } = require('@joplin/lib/locale');
 const { themeStyle } = require('../global-style.js');
 const Icon = require('react-native-vector-icons/Ionicons').default;
 const ModalDialog = require('../ModalDialog');
-const naturalCompare = require('string-natural-compare');
```

Comment:
Wait, but what's the history of this natural compare? Why are we using this? That's doing to introduce a regression, isn't it?

Reply:
Natural compare is an npm package which allows us to compare alphanumeric strings using alphanum algorithm. It was used to sort tags in the dialog window according to their numeric values. Not sure about the regression part. I tried sorting using localCompare on my machine and it worked.